rootfs/usr/lib/systemd/system/mkdir-rootfs@.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Enable mount points in / for ostree
+DefaultDependencies=no
+ConditionPathExists=!%f
+
+[Service]
```

Comment:
Ideally this entire unit would not even exist on non ostree distros.

Reply:
Maybe you could link it to the existence of `ostree-remount.service`?